### PR TITLE
add args to config

### PR DESCRIPTION
**Pull request by jrwren**

### CODE REVIEW

**Comment by jrwren** [comment]
for #4

File: encoder/encoder.go [hunk 1]
```diff
@@ -11,7 +11,10 @@ func encode(s string, d chan bool) {
 	fn := filepath.Base(s)
 	oname := fn[:strings.LastIndex(fn, ".")] + config.Format
 	log.Printf("Encoding Start: %s", s)
-	cmd := exec.Command("HandBrakeCLI", "-i", s, "-o", config.OutputDir+oname, "-Z", config.Preset, "--decomb=bob")
+	cmdargs := []string{"-i", s, "-o", config.OutputDir + oname,
+		"-Z", config.Preset, "--decomb=bob"}
```
Comment (ATTron) [inline encoder/encoder.go]: the ``` --decomb=bob ``` should be removed here as it will be handled by the ``` config.Args ``` call on line 16

**Review by ATTron — CHANGES_REQUESTED** [review]
Thanks for the help! If you could make this one small change I can merge this in. Would you also be able to update the README to include your changes? Specifically adding it to this section:

The config.json file takes the following arguments:
```
   WatchDir  <- The directory for Merlin to watch
   OutputDir <- Directory that Merlin will store output files
   Encoder   <- The chosen encoder format (run HandBrakeCLI --help to see list of available encoder formats)
   Preset    <- The HandBrakeCLI presets (run HandBrakeCLI -z to see available video presets)
   Format    <- The output format for the encoded files (av_mp4, av_mkv, av_webm)
```

**Review by ATTron — APPROVED** [review]

**Comment by ATTron** [comment]
@jrwren merged. Thanks!

**Pull request merged**